core/src/scene/importer.h

```diff
@@ -60,7 +61,7 @@ class Importer {
     // Imported scenes must be parsed into YAML nodes to find further imports.
     // The parsed scenes are stored in a map with their URLs to be merged once
     // all imports are found and parsed.
-    std::unordered_map<Url, Node> m_importedScenes;
+    std::unordered_map<Url, std::pair<Node, std::vector<Url>>> m_importedScenes = {};
```

Comment:
Can you update the documentation here?

Reply:
Also, a struct instead of a pair would make this much easier to read.